card/lib/card/migration.rb

```diff
@@ -62,12 +62,10 @@ def contentedly
     Card::Cache.reset_all
     Cardio.schema_mode "" do
       Card::Auth.as_bot do
-        ActiveRecord::Base.transaction do
```

Comment:
yes, they do and no, you added it in 2013 when there was no schema change:
https://www.evernote.com/l/ABLb922VeVlAcIh0-QhF6MPci2VUaQ53O8I